netmanager-app/package.json

```diff
@@ -24,6 +25,7 @@
     "@reduxjs/toolkit": "^2.5.0",
     "@tanstack/react-query": "^5.62.11",
     "@tanstack/react-query-devtools": "^5.62.11",
+    "@tanstack/react-table": "^8.20.6",
     "@types/leaflet": "^1.9.15",
```

Comment:
No need for a new table library, kindly use shadcn for all your UI needs! @danielmarv